system/include/simd128.h

```diff
@@ -0,0 +1,729 @@
+/*
+WebAssembly SIMD128 Intrinsics
+*/
+
+#include <stdint.h>
+
+// User-facing type
+typedef int32_t v128_t __attribute__((vector_size(16), __aligned__(16)));
```

Comment:
should be `__vector_size__` for consistency with other types

Reply:
Thank you!